src/App.js

```diff
@@ -578,103 +590,99 @@ export class App extends React.Component<AppProps, AppState> {
                             </Col>
                         </Row>
-                        <div className='App__playAndShare-container'>
-                            <div className='App__playControl-container'>
-                                <div className='App__playButton-container'>
-                                    <PlayButton
-                                        className='App__playButton'
-                                        interpreterIsRunning={this.state.runningState === 'running'}
-                                        disabled={this.state.programSequence.getProgramLength() === 0}
-                                        onClick={this.handleClickPlay}
+                    }
+                    */}
+                    <div className='App__command-palette'>
+                        <h2 className='App__command-palette-heading'>
+                            <FormattedMessage id='CommandPalette.movementsTitle' />
+                        </h2>
+                        <div className='App__command-palette-command-container'>
+                            <div className='App__command-palette-commands'>
+                                {this.renderCommandBlocks()}
+                            </div>
+                        </div>
+                    </div>
+                    <div className='App__scene-container'>
+                        <Scene
+                            dimensions={this.state.sceneDimensions}
+                            characterState={this.state.characterState}
+                            world={this.state.settings.world}
+                        />
+                        <div className='App__scene-controls'>
+                            <div className='App__scene-controls-group'>
+                                <PenDownToggleSwitch
+                                    className='App__penDown-toggle-switch'
+                                    value={this.state.drawingEnabled}
+                                    onChange={this.handleTogglePenDown}/>
+                                <div className='App__refreshButton-container'>
+                                    <RefreshButton
+                                        disabled={this.state.runningState === 'running'}
+                                        onClick={this.handleRefresh}
                                     />
-                                    <StopButton
-                                        disabled={
-                                            this.state.runningState === 'stopped'
-                                            || this.state.runningState === 'stopRequested'}
-                                        onClick={this.handleClickStop}/>
                                 </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div className="App__world-selector-container">
+                        <WorldSelector onSelect={this.handleChangeWorld} />
```

Comment:
Let's pass world state to WorldSelector.

`
world={this.state.setting.world}
`

Reply:
Done.